data-hibernate-jpa/src/test/java/io/micronaut/data/hibernate/entities/EventIndividualTest.java

```diff
@@ -17,6 +17,7 @@ public class EventIndividualTest {
     @GeneratedValue
     private Long id;
     @AutoPopulated
+    @Column(columnDefinition = "uuid")
```

Comment:
This is needed or else column is mapped to byte(255) or something like that which fails